templates/phoenix_elixir/files/.platform.app.yaml

```diff
@@ -0,0 +1,42 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses.
+type: elixir:1.9
+
+variables:
+    env:
+        SECRET_KEY_BASE: $PLATFORM_PROJECT_ENTROPY
```

Comment:
I had issue with last phoenix release I was force to have a 64 chars length here and I think PLATFORM_PROJECT_ENTROPY is 56 chars currently, should we let the user change it or let it fail ?

Reply:
Laravel has a similar problem.  What we do there (albeit in PHP space) is compute a proper-length hash using the entropy value as a seed.

cf: https://github.com/platformsh/laravel-bridge/blob/master/platformsh-laravel-env.php#L30

Might something similar-ish work here?

Reply:
I don't know that much Laravel but yes look like we could do something like this, don't know how but its interesting topic